README.md

```diff
@@ -156,10 +156,25 @@ The Julia launcher `julia` automatically determines which specific version of Ju
 1. A command line Julia version specifier, such as `julia +release`.
 2. The `JULIAUP_CHANNEL` environment variable.
 3. A directory override, set with the `juliaup override set` command.
-3. The default Juliaup channel.
+4. Automatic version selection based on the active project.
```

Comment:
Can you make this a clickable link to the "Project-based Version Selection" section below?